File: app/src/main/java/com/jennifer/andy/simpleeyes/player/event/VideoProgressEvent.java
package com.jennifer.andy.simpleeyes.player.event;

/**
 * Author:  andy.xwt
 * Date:    2018/3/13 18:10
 * Description:
 */


public class VideoProgressEvent {

    private int duration;
    private int currentPosition;
    private int progress;
    private int secondaryProgress;

    public VideoProgressEvent(int duration, int currentPosition, int progress, int secondaryProgress) {
        this.duration = duration;
        this.currentPosition = currentPosition;
        this.progress = progress;
        this.secondaryProgress = secondaryProgress;
    }

    public int getDuration() {
        return duration;
    }

    public void setDuration(int duration) {
        this.duration = duration;
    }

    public int getCurrentPosition() {
        return currentPosition;
    }

    public void setCurrentPosition(int currentPosition) {
        this.currentPosition = currentPosition;
    }

    public int getProgress() {
        return progress;
    }

    public void setProgress(int progress) {
        this.progress = progress;
    }

    public int getSecondaryProgress() {
        return secondaryProgress;
    }

    public void setSecondaryProgress(int secondaryProgress) {
        this.secondaryProgress = secondaryProgress;
    }
}

File: app/src/main/java/com/jennifer/andy/simpleeyes/player/event/VideoProgressEvent.kt
package com.jennifer.andy.simpleeyes.player.event


/**
 * Author:  andy.xwt
 * Date:    2020/3/9 10:02 PM
 * Description: 视频播放进度事件
 */

data class VideoProgressEvent(
        var duration: Int,//总时间
        var currentPosition: Int,//当前播放时间
        var progress: Int,//第一进度（播放进度）
        var secondaryProgress: Int//第二进度（下载进度）
)
